File: VCL/Delphi/SimpleEdit/InfoForm.pas
//=============================================================================
// This source code is a part of TatukGIS Developer Kernel.
// (c)2000-2025 TatukGIS. ALL RIGHTS RESERVED.
//=============================================================================
{
  How to provide editing functionality.
}
unit InfoForm;

interface

uses
  System.SysUtils,
  System.Classes,
  System.Variants,

  Vcl.Forms,
  Vcl.StdCtrls,
  Vcl.Buttons,
  Vcl.Controls,
  Vcl.Grids,
  Vcl.ExtCtrls,

  GisLayerVector,
  GisUtils,
  GisViewer,

  Vcl.GisControlAttributes;

type
  TfrmInfo = class(TForm)
    GISAttributes: TGIS_ControlAttributes;

  public
    { Public declarations }
    procedure ShowInfo( const _shp : TGIS_Shape ) ;
  end;

var
  frmInfo: TfrmInfo;

implementation

{$R *.DFM}


procedure TfrmInfo.ShowInfo( const _shp : TGIS_Shape ) ;
begin
  GISAttributes.ShowShape( _shp ) ;
end ;

end.

File: VCL/C++Builder/SimpleEdit/InfoForm.cpp
//=============================================================================
// This source code is a part of TatukGIS Developer Kernel.
//=============================================================================
//
//  How to provide editing functionality.
//
//  Check project\options\directories in a case of any problems during compilation
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "InfoForm.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma resource "*.dfm"
TfrmInfo *frmInfo;
//---------------------------------------------------------------------------
__fastcall TfrmInfo::TfrmInfo(TComponent* Owner)
        : TForm(Owner)
{
}
//---------------------------------------------------------------------------

void TfrmInfo::ShowInfo( TGIS_Shape * const _shp )
{
  GISAttributes->ShowShape( _shp ) ;
}
//---------------------------------------------------------------------------




